chain/actors/builtin/evm/evm.go

```diff
@@ -16,6 +17,17 @@ import (
 
 var Methods = builtin12.MethodsEVM
 
+const (
+	ErrReverted exitcode.ExitCode = iota + 33
+	ErrInvalidInstruction
+	ErrUndefinedInstruction
+	ErrStackUnderflow
+	ErrStackOverflow
+	ErrIllegalMemoryAccess
+	ErrBadJumpdest
+	ErrSelfdestructFailed
+)
```

Comment:
where are these used?

Reply:
https://github.com/filecoin-project/lotus/blob/cf910a8d1a25d46267412b3e5f7a569ef16b592d/node/impl/full/eth_trace.go#L142-L159

Reply:
They're returned by the EVM on error: https://github.com/filecoin-project/builtin-actors/blob/6e781444cee5965278c46ef4ffe1fb1970f18d7d/actors/evm/src/lib.rs#L35-L42